src/include/txn.h

```diff
@@ -176,14 +176,27 @@ struct __wt_txn_op {
 	enum {
```

Comment:
@agorrod, I will check this, but file id using also seems to be feasible, as recovery is using the file id only.

Reply:
@bvpvamsikrishna you're correct that recovery works this way (because btree IDs are what we write into the log), but that depends on recovery maintaining a mapping from btree IDs to cursors, which doesn't exist at runtime.